### PR TITLE
Javascript module milestone

### PR DESCRIPTION
**Pull request by assadounto**
In this milestone 

- added luxon library to display time
- refactored awesome books
- Used javascript modules to implement functionality

### CODE REVIEW

File: modules/functions.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
+const booksLibrary = document.querySelector('#booksLibrary');
+const form = document.querySelector('#newBook');
+const booksTable = document.querySelector('#booksTable');
+const contact = document.querySelector('#contact');
+export function colorCards() {
```
Comment (rloterh) [inline modules/functions.js]: - [x] As per the project requirement kindly ensure to **replace all regular functions with arrow functions**

File: modules/classmethods.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import * as functions from './functions.js';
+
+const title = document.querySelector('#title');
+const author = document.querySelector('#author');
+let myLibrary = [];
+// const storage = [];
```
Comment (rloterh) [inline modules/classmethods.js]: - [x] OPTIONAL
There is a commented code block found in your code on the repository. Kindly endeavor to remove **all** commented code from your code before pushing it to the repository so as to keep your code well organized.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Rich, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Descriptive Pull Request
- Module import correctly implemented
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly check the comments marked **[OPTIONAL]** under the review._

You can also consider:
N/A

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Rich, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**